docs/compute-resources.md

```diff
@@ -68,26 +68,26 @@ in a `Task's` pod, unless there is a LimitRange present in the namespace.
 (Tekton doesn't allow users to configure init containers for a `Task`.)
 Tekton supports LimitRange minimum, maximum, and default resource requirements for containers,
 but does not support LimitRange ratios between requests and limits ([#4230](https://github.com/tektoncd/pipeline/issues/4230)).
-LimitRange types other than "Container" are not supported.
+LimitRange types other than "Container" are not considered for purposes of resource requirements.
 
 ### Requests
 
-If a `Step` does not have requests defined, the resulting container's requests are the larger of:
+If a container does not have requests defined, the resulting container's requests are the larger of:
 - the LimitRange minimum resource requests 
-- the LimitRange default resource requests, divided among the app containers
+- the LimitRange default resource requests (for init and Sidecar containers) or the LimitRange default resource requests divided by the number of Step containers (for Step containers)
```

Comment:
thank you for clarifying this 👍

Reply:
Lee's new and improved verbiage not mine ;)